README.md

```diff
@@ -21,15 +21,16 @@ This Git repository stores information about NumFOCUS'
 application for Google Summer of Code in the current and previous years.
 
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
+
 **Table of Contents**
 
 - [Students](#students)
 - [Sub Organizations](#sub-organizations)
 - [Organizations Confirmed Under NumFOCUS Umbrella](#organizations-confirmed-under-numfocus-umbrella)
 - [NumFOCUS Projects](#numfocus-projects)
-    - [Fiscally Sponsored Projects GSoC Status](#fiscally-sponsored-projects-gsoc-status)
-    - [Affiliated Projects GSoC Status](#affiliated-projects-gsoc-status)
-    - [Other Projects GSoC Status](#other-projects-gsoc-status)
+  - [Fiscally Sponsored Projects GSoC Status](#fiscally-sponsored-projects-gsoc-status)
+  - [Affiliated Projects GSoC Status](#affiliated-projects-gsoc-status)
```

Comment:
Yeah, the auto formatting breaks markdown.

Reply:
I'll close and redo it, thanks!

Reply:
Thanks for taking the time to fix it up!